src/containers/Home.jsx

```diff
@@ -1,11 +1,14 @@
 import React, { Component } from 'react';
 import { Container } from 'react-bootstrap';
 
+import VideoFeed from './VideoFeed';
+
 class Home extends Component {
   render() {
     return (
       <Container className="home-container" fluid>
         <h1>Home Page</h1>
+        <VideoFeed numVideos="3" owner="36" />
```

Comment:
Why are these values hardcoded?

Reply:
Owner prop should not be passed in the Home.jsx. Only in for the Profile.jsx.

This is there for now, since otherwise the VideoFeed component won't load any videos due to the private file access as said

Reply:
Changed the numVideos props to an optional route prop for Home.jsx instead.
owner should still be taken away when we have the private file access fixed